pset-functions.js

```diff
@@ -31,6 +31,40 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+//return the card score of either player that is less than or equal to 21
+//difference of 21 - card score
+
+let playerCardScore = 0
+let dealerCardScore = 0
+let result = 0
+
+function blackJack(playerCardScore,dealerCardScore){
+  if (playerCardScore > 21 && dealerCardScore > 21){
+    result = 0
+  } else if(playerCardScore === 21 && dealerCardScore === 21) {
+    result = 21
+  } else if(playerCardScore > 21) {
+    result = dealerCardScore
+  } else if(dealerCardScore > 21) {
+    result = playerCardScore
+  } else if ((21 - playerCardScore) < (21-dealerCardScore)){
```

Comment:
These else/if statements work perfectly well; but here is an alternative solution to add to your tool-belt that takes advantage of javascript's built in Math methods [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L41)